pkg/apimachinery/patch/patch.go

```diff
@@ -140,6 +140,21 @@ func GeneratePatchPayload(patches ...PatchOperation) ([]byte, error) {
 	return payloadBytes, nil
 }
 
+func (p *PatchSet) ToSlice() ([]string, error) {
+	var result []string
+
+	for _, operation := range p.GetPatches() {
```

Comment:
```suggestion
	for _, operation := range p.patches {
```

nit: For consistency with other methods

Reply:
done